docs/preservation-actions.md

```
@@ -131,14 +131,14 @@ After approving the triage:
 1. Exit the sandbox.
 2. Fetch the branch from the sandbox:
    ```sh
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)